sinfonier-web/app/models/storm/ParamsValidator.java

```diff
@@ -64,7 +64,7 @@ public boolean hasParams(String key) {
 
   public boolean validate(String key, Object value) throws SinfonierException {
     if (!hasParams(key)) {
-      return false;
+      return (validateExtraConfKey(key) && validateExtraConfValue(((String) value).trim()));
```

Comment:
The only actual call to this method is from public boolean validate(TopologyConfig config).
In this method we call parseExtraConfig, that throws an exception if there are a config key without value.
Maybe this method must be private.